data_steward/cdr_cleaner/cleaning_rules/base_cleaning_rule.py

```diff
@@ -129,6 +169,7 @@ def __init__(self,
         self._sandbox_dataset_id = sandbox_dataset_id
         self._issue_urls = issue_urls if issue_urls else []
         self._depends_on_classes = depends_on if depends_on else []
+        self.TABLE_COUNT_QUERY = ''' SELECT COALESCE(COUNT(*), 0) AS row_count FROM `{dataset}.{table}` '''
```

Comment:
adding {project} is a good idea. I'll make the change.

I used coalesce because when the tables are empty we get none object as a return value so to make it easy to compute I made it pass 0 as row count when the table is empty.

Reply:
Nice.  I had not thought about using coalesce for that.  I like it.

Reply:
`TABLE_COUNT_QUERY` feels like a class variable instead of an instance variable.